config/analytics_hidden_pii.yml

```diff
@@ -0,0 +1,15 @@
+---
+shared:
+  application_forms:
+    - support_reference
+    - candidate_id
+  candidates:
+    - id
+  support_users:
+    - dfe_sign_in_uid
+  provider_users:
+    - dfe_sign_in_uid
+  validation_errors:
+    - user_id
+  emails:
+    - subject
```

Comment:
I aligned with Data insights team that this table is large, the script on their side to migrate might take a while to finish.